MAiD - Case Management App/force-app/main/default/reportTypes/MMHA_Reporting_Report_Type.reportType-meta.xml

```diff
@@ -0,0 +1,1198 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ReportType xmlns="http://soap.sforce.com/2006/04/metadata">
+    <baseObject>Case</baseObject>
```

Comment:
Has this list of available fields been validated against UAT? @lilianenassar-bib A good source to match might be the permission set for the ICY Integrate app for fields to include as this is including some MAiD fields (not crucial, as they won't see them, but could be cleared up now instead of later).

Reply:
All Case Object Fields has been reviewed with source github for all reports Types that use Case Object not to have conflict on deployment @cgijeffolsen